src/sql/azdata.d.ts

```diff
@@ -100,6 +100,8 @@ declare module 'azdata' {
 			password: string;
 			authenticationType: string;
 			savePassword: boolean;
+			host: string;
```

Comment:
1. Never add stuff like this directly to azdata.d.ts, always add it to azdata.proposed.d.ts first
1. I'm not sure about adding these in the first place - this stuff should be general across all types of connections where as this seems to be something specifically used by PGSQL. Or was this being done elsewhere and just not hooked up everywhere?